jzp-code/kukdo/project/cloud-storage/cloud-storage-parent/cloud-storage-web/src/main/java/com/xiaomi/unfin/CloudStorageClient.java

```diff
@@ -0,0 +1,124 @@
+package com.xiaomi.unfin;
+
+import org.apache.hc.client5.http.classic.methods.*;
+import org.apache.hc.client5.http.entity.UrlEncodedFormEntity;
+import org.apache.hc.client5.http.entity.mime.MultipartEntityBuilder;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
+import org.apache.hc.client5.http.impl.classic.HttpClients;
+import org.apache.hc.core5.http.ParseException;
+import org.apache.hc.core5.http.io.entity.EntityUtils;
+import org.apache.hc.core5.http.message.BasicNameValuePair;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+@Command(name = "cloudstorage", mixinStandardHelpOptions = true, version = "1.0",
+        description = "Client for interacting with cloud storage service")
+public class CloudStorageClient implements Runnable {
+
+    @Option(names = {"-u", "--upload"}, description = "Upload file to cloud storage")

```

Comment:
use a sub-command is better, like:
```
my-cloud-storage upload --name <file-name> --file <client-file-path>
my-cloud-storage download --name <file-name> --destination <client-file-path>
my-cloud-storage delete --name <file-name>
my-cloud-storage list
```

Reply:
> use a sub-command is better, like:
> 
> ```
> my-cloud-storage upload --name <file-name> --file <client-file-path>
> my-cloud-storage download --name <file-name> --destination <client-file-path>
> my-cloud-storage delete --name <file-name>
> my-cloud-storage list
> ```

I'll fix the bugs of this part and implement it based on your productive suggestions, the new PR is expected on 6.25